lib/dashboard/home/tab_bar/tokens/confirm_withdrawal/cubit/confirm_withdrawal_state.dart

```diff
@@ -0,0 +1,39 @@
+part of 'confirm_withdrawal_cubit.dart';
+
+@JsonSerializable()
+class ConfirmWithdrawalState extends Equatable {
+  const ConfirmWithdrawalState({
```

Comment:
In confirm withdrawal page, withdrawAddress, selectedToken and amount are static. I don't think we need state for that...